bundles/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/internal/handler/KeContactHandler.java

```diff
@@ -94,30 +97,49 @@ public KeContactHandler(Thing thing, KeContactTransceiver transceiver) {
 
     @Override
     public void initialize() {
-        if (getConfig().get(IP_ADDRESS) != null && !getConfig().get(IP_ADDRESS).equals("")) {
-            transceiver.registerHandler(this);
-
-            cache = new ExpiringCacheMap<>(
-                    Math.max((((BigDecimal) getConfig().get(POLLING_REFRESH_INTERVAL)).intValue()) - 5, 0) * 1000);
-
-            cache.put(CACHE_REPORT_1, () -> transceiver.send("report 1", getHandler()));
-            cache.put(CACHE_REPORT_2, () -> transceiver.send("report 2", getHandler()));
-            cache.put(CACHE_REPORT_3, () -> transceiver.send("report 3", getHandler()));
-            cache.put(CACHE_REPORT_100, () -> transceiver.send("report 100", getHandler()));
-
-            if (pollingJob == null || pollingJob.isCancelled()) {
-                try {
-                    pollingJob = scheduler.scheduleWithFixedDelay(this::pollingRunnable, 0,
-                            ((BigDecimal) getConfig().get(POLLING_REFRESH_INTERVAL)).intValue(), TimeUnit.SECONDS);
-                } catch (Exception e) {
-                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE,
-                            "An exception occurred while scheduling the polling job");
+        logger.debug("Initializing Keba binding");
+        try {
+            if (isKebaReachable()) {
+                transceiver.registerHandler(this);
+
+                cache = new ExpiringCacheMap<>(
+                        Math.max((((BigDecimal) getConfig().get(POLLING_REFRESH_INTERVAL)).intValue()) - 5, 0) * 1000);
+
+                cache.put(CACHE_REPORT_1, () -> transceiver.send("report 1", getHandler()));
+                cache.put(CACHE_REPORT_2, () -> transceiver.send("report 2", getHandler()));
+                cache.put(CACHE_REPORT_3, () -> transceiver.send("report 3", getHandler()));
+                cache.put(CACHE_REPORT_100, () -> transceiver.send("report 100", getHandler()));
+
+                if (pollingJob == null || pollingJob.isCancelled()) {
+                    try {
+                        pollingJob = scheduler.scheduleWithFixedDelay(this::pollingRunnable, 0,
+                                ((BigDecimal) getConfig().get(POLLING_REFRESH_INTERVAL)).intValue(), TimeUnit.SECONDS);
+                    } catch (Exception e) {
+                        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE,
+                                "An exception occurred while scheduling the polling job");
+                    }
                 }
+            } else {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
+                        "IP address or port number not set");
             }
-        } else {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
-                    "IP address or port number not set");
+        } catch (Exception e) {
+            logger.error("Exception during Ping check and Http Backup check", e);
```

Comment:
Bindings should only log to error if something severe happened, like the detection of a bug in your code. This could be debug. See this link for a description of the log levels: https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
Ack. Changed  to debug.